google/cloud/opentelemetry/internal/recordable.h

```diff
@@ -27,6 +27,9 @@ namespace cloud {
 namespace otel_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
+// https://github.com/googleapis/googleapis/blob/a5e03049afc43ccb1217d5dd3be53006fc278643/google/devtools/cloudtrace/v2/trace.proto#L233
+std::size_t constexpr kDisplayNameStringLimit = 128;
```

Comment:
It sure would be nice if this limit was in a constant (an enum?) in the proto file instead of just a comment.

Reply:
No opinion.